multi-node/aws/pkg/config/templates/stack-template.json

```diff
@@ -256,7 +256,11 @@
           {
             "DeviceName": "/dev/xvda",
             "Ebs": {
-              "VolumeSize": "{{.ControllerRootVolumeSize}}"
+              "VolumeSize": "{{.ControllerRootVolumeSize}}",
+              {{if gt .ControllerRootVolumeIOPS 0}}
```

Comment:
@colhom I just came up with [this](https://github.com/coreos/coreos-kubernetes/pull/559/commits/e320532aef62014c162b9102919d6c3564ba6d9b), so now there's a test to check that volume type and iops options are getting rendered correctly in the final stack template.

I'm not a Golang expert, so there might be a better way to write this code. Let me know if you have a better idea.